ratis-server-api/src/main/java/org/apache/ratis/server/RaftServerConfigKeys.java

```diff
@@ -457,6 +457,18 @@ static void setRetentionFileNum(RaftProperties properties, int numSnapshotFilesR
   interface DataStream {
     String PREFIX = RaftServerConfigKeys.PREFIX + ".data-stream";
 
+    String USE_CACHED_THREAD_POOL_KEY = PREFIX + ".use.cached.thread.pool";
```

Comment:
Let's change it to ".async.request.thread.pool.cached".  Then, it is easy to see that this conf and ".async.request.thread.pool.size" are closely related.